android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeet.java

```diff
@@ -42,7 +42,7 @@ public static void setDefaultConferenceOptions(JitsiMeetConferenceOptions option
      * @return the current conference URL.
      */
     public static String getCurrentConference() {
-        return OngoingConferenceTracker.getCurrentConference();
+        return OngoingConferenceTracker.getInstance().getCurrentConference();
```

Comment:
I don't know when this method was added, but it basically doesn't care about the scope being a static method. Maybe we should remove the scope and say SDK supports only one activity at a time then ?